src/Assignments/SimpleProblemPage.tsx

```diff
@@ -44,13 +45,27 @@ export const SimpleProblemPage: React.FC<SimpleProblemPageProps> = () => {
     const [modalLoading, setModalLoading] = useState<boolean>(false);
     const [selectedProblemId, setSelectedProblemId] = useState<number | null>(null);
     const [loading, setLoading] = useState(true);
-    const [error, setError] = useState('');
+    const [alert, setAlert] = useAlertState();
     const [confirmationParameters, setConfirmationParameters] = useState<ConfirmationModalProps>(DEFAULT_CONFIRMATION_PARAMETERS);
     const [openDrawer, setOpenDrawer] = useState<boolean>(false);
     const {course, users} = useCourseContext();
 
+    const resetAlert = (): void => {
+        setAlert({
```

Comment:
I like this, but we should statically store the object (with a `useRef` or so) this is using so that we avoid unnecessary rerenders.